executor/src/state.rs

```diff
@@ -3,10 +3,6 @@
 use serde::{Deserialize, Serialize};
 use types::SequenceNumber;
 
-#[cfg(test)]
-#[path = "tests/state_tests.rs"]
-pub mod state_tests;
-
```

Comment:
Since you removed the tests I would recommend to remove the implementation methods of `ExecutionIndices` as well since they don't seem to be used anymore.